docker/k8s/README.md

```diff
@@ -45,27 +46,31 @@ in SGX-SIM and SGX-HW mode.
     ```
 
     > The above command would run the 
-    > [propose_requests.sh](./scripts/propose_requests.sh) as a kubernetes job
+    > [propose_requests.sh](../../scripts/propose_requests.sh) as a kubernetes job
 
 3. Check the request handling logs (the request should be done in several 
    minutes after the proposal)
     ```bash
-    # Let's say 'propose-request-abcde' is the job created above
+    # Get workload request pod name using command
+    kubectl get pods
+    # Let's say 'propose-request-abcde' is the job pod created above. Check the logs
     kubectl logs propose-request-abcde
 
     # The partial output for the last query should be go as below
     [23:20:36 INFO    __main__] Signature Verified
-    [23:20:36 INFO    utility.utility] Decryption Result at Client - You have a risk of 71% to have heart disease. 
+    [23:20:36 INFO    utility.utility] Decryption Result at Client - You have a risk of 47% to have heart disease.
     ```
 
 ## Create Avalon shell pod
 1. Avalon shell pod can be useful to run applications manually.
     ```bash
     kubectl create -f shell.yaml
     ```
-2. After verifying avalon shell pod is running get a shell to the running container
+2. After verifying avalon shell pod is running, get a shell to the running container
     ```bash
-    # Let's say 'avalon-shell-abcde' is the pod created above
+    # Get avalon shell pod name using command
```

Comment:
Change "Get avalon shell pod name" to "Get the Avalon shell pod name"

Reply:
Done